src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/HotReload/Contracts/IProjectHotReloadAgent.cs

```diff
@@ -1,28 +1,28 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
+using System.Runtime.CompilerServices;
 using Microsoft.VisualStudio.ProjectSystem.Debug;
-using Microsoft.VisualStudio.ProjectSystem.HotReload;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload;
 
+[InternalImplementationOnly]
 [ProjectSystemContract(ProjectSystemContractScope.ProjectService, ProjectSystemContractProvider.System, Cardinality = ImportCardinality.ExactlyOne)]
 public interface IProjectHotReloadAgent
 {
-    IProjectHotReloadSession? CreateHotReloadSession(string id, int variant, string runtimeVersion, IProjectHotReloadSessionCallback callback);
```

Comment:
Let's still keep these APIs for now and remain backward compatible? There are some internal projects (including webtools) consumes these APIs.

Reply:
I removed all internal usage.